style.css

```diff
@@ -70,21 +78,21 @@ p {
 }
 
 .Langlist {
-  background-image: url(/imgs/Ellipse19.svg);
+  background-image: url(/imgs/Ellipse19left.svg);
   background-repeat: no-repeat;
   background-position: right;
```

Comment:
Kindly modify the alignment of the `main-section` to match the figma design.

Sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now. You'll get more prepared for a typical workplace experience where you might be required to reproduce a given design as strictly as possible.

So, let's follow the original vision if it's pretty and works, it will be a wonderful website.

Your design
![image](https://user-images.githubusercontent.com/20554583/186680380-7f86739e-1be9-4179-a16b-e6929554e6f1.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/186681052-4ef8d86b-81a3-432d-991e-c770fa3ea1fc.png)